src/app/external-cluster-wizard/steps/external-cluster/provider/eks/component.ts

```diff
@@ -175,16 +197,19 @@ export class EKSClusterSettingsComponent
         .valueChanges.pipe(debounceTime(this._debounceTime))
         .pipe(takeUntil(this._unsubscribe))
         .subscribe(_ => {
-          const vpc = this.controlValue(Controls.Vpc)?.main;
+          this._clearSubnet();
+          this._clearSecurityGroups();
```

Comment:
I think we should not call this method in dialog view because security groups element doesn't exist in dialog.

Reply:
No, since it is under `else` condition which will be executed only in wizard case.